src/main/resources/assets/vulkanmod/lang/ru_ru.json

```
@@ -0,0 +1,40 @@
+{
+  "vulkanmod.options.unknown": "Неизвестно",
+
+  "vulkanmod.options.pages.video": "Видео",
+  "vulkanmod.options.pages.graphics": "Графика",
+  "vulkanmod.options.pages.optimizations": "Оптимизация",
+  "vulkanmod.options.pages.other": "Другое",
+
+  "vulkanmod.options.buttons.apply": "Применить",
+  "vulkanmod.options.buttons.kofi": "Поддержите меня",
+
+  "vulkanmod.options.advCulling": "Отбраковка чанков",
+  "vulkanmod.options.advCulling.aggressive": "Агрессивная",
+  "vulkanmod.options.advCulling.conservative": "Сниженная",
+  "vulkanmod.options.advCulling.normal": "Обычная",
+  "vulkanmod.options.advCulling.tooltip": "Алгоритм отбраковкий, который может улучшить производительность благодаря уменьшению количества отображаемых чанков.",
+
+  "vulkanmod.options.ao.subBlock": "Вкл (Полублок)",
+  "vulkanmod.options.ao.subBlock.tooltip": "Вкл (Полублок): Мягкое освещение на неполных блоках (экспериментально).",
+
+  "vulkanmod.options.deviceSelector": "Выбор устройства",
+  "vulkanmod.options.deviceSelector.auto": "Авто",
+  "vulkanmod.options.deviceSelector.tooltip": "Текущее устройство",
+
+  "vulkanmod.options.entityCulling": "Отбраковка сущностей",
+  "vulkanmod.options.entityCulling.tooltip": "Отбраковывает сущностей в не видимых игроку областях.",
+
+  "vulkanmod.options.frameQueue": "Размер очереди рендеринга",
+  "vulkanmod.options.frameQueue.tooltip": "Более высокие значения могут помочь стабилизировать время между кадрами, но увеличат задержку ввода.",
```

Предлагаю сократить (+ `время кадра` – это термин)

```suggestion
  "vulkanmod.options.frameQueue.tooltip": "Более высокие значения могут улучшить стабильность времени кадра, но увеличат задержку ввода.",
```